### PR TITLE
Update intro-to-inverse-functions

### PR DESCRIPTION
**Pull request by bobprinceraj**
**Description**

### CODE REVIEW

File: src/pages/mathematics/intro-to-inverse-functions/index.md [hunk 1]
```diff
@@ -2,8 +2,14 @@
 title: Intro to Inverse Functions
 ---
 ## Intro to Inverse Functions
+  <p>Say you were trying to reflect y=3xover the line y=x. What would be the equation of the line? As you can see in the image below, it would be y=1/3x. But why is that so? If you swap x and y with each other and solve for y, you can find that the equation would be the same. This is what would be called a inverse function.
+<img src = "http://d2r5da613aq50s.cloudfront.net/wp-content/uploads/369430.image1.jpg" />
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/intro-to-inverse-functions/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+
+###Sources:
+  “How to Graph the Inverse of a Function.” Dummies, www.dummies.com/education/math/calculus/how-to-graph-the-inverse-of-a-function/.
```
Comment (1andee) [inline src/pages/mathematics/intro-to-inverse-functions/index.md]: This link isn't formatted properly. Check [here](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#links) for guidance on how to do this in Markdown.

File: src/pages/mathematics/intro-to-inverse-functions/index.md [hunk 2]
```diff
@@ -2,8 +2,14 @@
 title: Intro to Inverse Functions
 ---
 ## Intro to Inverse Functions
+  <p>Say you were trying to reflect y=3xover the line y=x. What would be the equation of the line? As you can see in the image below, it would be y=1/3x. But why is that so? If you swap x and y with each other and solve for y, you can find that the equation would be the same. This is what would be called a inverse function.
+<img src = "http://d2r5da613aq50s.cloudfront.net/wp-content/uploads/369430.image1.jpg" />
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/mathematics/intro-to-inverse-functions/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+
+###Sources:
```
Comment (1andee) [inline src/pages/mathematics/intro-to-inverse-functions/index.md]: You'll want to add a space between the number signs and heading for this to properly display.

File: src/pages/mathematics/intro-to-inverse-functions/index.md [hunk 3]
```diff
@@ -2,8 +2,14 @@
 title: Intro to Inverse Functions
 ---
 ## Intro to Inverse Functions
+  <p>Say you were trying to reflect y=3xover the line y=x. What would be the equation of the line? As you can see in the image below, it would be y=1/3x. But why is that so? If you swap x and y with each other and solve for y, you can find that the equation would be the same. This is what would be called a inverse function.
```
Comment (1andee) [inline src/pages/mathematics/intro-to-inverse-functions/index.md]: No need for a paragraph tag in Markdown.

**Comment by BerkeleyTrue** [comment]
Deploy preview failed.

Built with commit 7bb98e5e80419ffdbc4baabed95e7adfcc75c093

https://app.netlify.com/sites/guide/deploys/5a1c63454c4b93377865ba4b

**Pull request merged**